db/migrate/20161130152451_create_admin_admission_process_image_details.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAdminAdmissionProcessImageDetails < ActiveRecord::Migration[5.0]
+  def change
+    create_table :admin_admission_process_image_details do |t|
+      #t.references :admission_process_image, foreign_key: true
+      t.text :details
+      t.references :admission_process_image, foreign_key: true
```

Comment:
I don't  know why you don't follow my instruction, please fix this error as soon as possible.